src/core/net/nat64_translator.hpp

```diff
@@ -163,7 +163,87 @@ class Translator : public InstanceLocator, private NonCopyable
      */
     void SetNat64Prefix(const Ip6::Prefix &aNat64Prefix);
 
+    /**
+     * @brief This function gets the next AddressMapping info (using an iterator).
```

Comment:
No need for`@brief` (first line is treated as it automatically by doxygen). 

Also "This function" -> "This method"

Reply:
Removed "This function" (and other "this function/method")